src/main/java/com/revature/controllers/ContentController.java

```diff
@@ -97,19 +97,27 @@
 	// parameters passed in
 	// If a parameter is empty, it is not used in the filtering process.
 	// modules is a string in comma separated format of integers ex. "1,2,3,4"
+	
+	//Change this to tokenize the format input the same way we do modules
 	@LogException
 	@GetMapping(params = { "title", "format", "modules" })
 	public ResponseEntity<Set<JSONContent>> getSearchResults(
 			@RequestParam(value = "title", required = false) String title,
 			@RequestParam(value = "format", required = false) String format,
 			@RequestParam(value = "modules", required = false) String modules) {
 		ArrayList<Integer> moduleIdsList = new ArrayList<Integer>();
+		ArrayList<String> formatList = new ArrayList<String>();
 		StringTokenizer st = new StringTokenizer(modules, ",");
-		while (st.hasMoreTokens()) {
+		while(st.hasMoreTokens()) {
 			moduleIdsList.add(Integer.parseInt(st.nextToken()));
 		}
+		
+		StringTokenizer ft = new StringTokenizer(format, ",");
+		while (ft.hasMoreTokens()) {
+			formatList.add(ft.nextToken());
+		}
 
-		Set<Content> contentSet = searchService.filter(title, format, moduleIdsList);
+		Set<Content> contentSet = searchService.filter(title, formatList, moduleIdsList);
```

Comment:
Why use a tokenizer here rather than just map the Request Param into an ArrayList?

Reply:
Also applicable for RequestController

Reply:
To maintain the convention used by previous team, they used tokenizer for Module

Reply:
As for why they chose to do that, I can't answer.

Reply:
I would refactor this; spring will do it for you

Reply:
Will do

Reply:
So I looked into this and the way the application is currently built, spring will not do it automatically in @RequestParam()

In order for RequestParam to catch an array of objects, the format of the request has to group the array like this.

param=value1&param=value2&param=value3

while our application currently passes arrays/lists like so

param=value1,value2,value3

That being said, the need for tokenizer is still not required as we can simply use the String.split(",") method to get our array.  I can of course change both ends to change the structure of how our request params are passed, but I think the split method would entail less gutting.  If your ok with it, that's the refactor I'd like to do.

Reply:
Nevermind, I found the documentation that allows for it in requestparam.
I'm an idiot.

Reply:
Fixed and pushed.